<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upnextElement</class>
 <widget class="QWidget" name="upnextElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>105</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>102</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(105, 255, 160);
border-radius:15px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="taskName">
          <property name="minimumSize">
           <size>
            <width>570</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 26pt &quot;Cascadia Code&quot;;</string>
          </property>
          <property name="text">
           <string>LOrem Ipsum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="taskType">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font:  &quot;Cascadia Code&quot;;</string>
          </property>
          <property name="text">
           <string>NO type!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="deadline">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 15pt &quot;Cascadia Code&quot;;</string>
          </property>
          <property name="text">
           <string>Boom</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="rmDays">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 15pt &quot;Cascadia Code&quot;;
background:rgb(105, 255, 160);
border-radius:15px;</string>
          </property>
          <property name="text">
           <string>Dead</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 15pt &quot;Cascadia Code&quot;;</string>
          </property>
          <property name="text">
           <string>Deadline on:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 15pt &quot;Cascadia Code&quot;;</string>
          </property>
          <property name="text">
           <string>Remaining days:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="modButtons">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: #fff;
  border: 0 solid #e2e8f0;
  border-radius: 5px;
  color: #0d172a;
  font-family: &quot;Basier circle&quot;,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
  font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 5px 5px;
  text-align: center;
  text-decoration:none #0d172a solid;
}

QPushButton:hover {
  background-color: #221D23;
  color: #fff;
}
</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="editTaskButton">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_taskButton">
           <property name="styleSheet">
            <string notr="true">
QPushButton:hover {
  background-color: red;
  color: #fff;
}
</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
